nova_plugin/tests/resources/test-keypair-validation-blueprint.yaml

```diff
@@ -0,0 +1,25 @@
+
+tosca_definitions_version: cloudify_dsl_1_2
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/3.3.1/types.yaml
+  - plugin.yaml
+
+inputs:
+  private_key:
+    default: ''
+  is_keypair_external:
+    default: false
+
+node_templates:
+
+  keypair:
+    type: cloudify.openstack.nodes.KeyPair
+    properties:
+      private_key_path: {get_input: private_key}
```

Comment:
we generally write intrinsic function with spaces in the edges
`{ get_input: private_key }`